invenio_orcid/utils.py

```diff
@@ -31,3 +31,30 @@ def get_authors_credentials(orcid):
         id=orcid, method='orcid').first()
     user = RemoteAccount.query.filter_by(user_id=raw_user.id_user).first()
     return user.tokens[0].access_token
+
+
+def convert_to_orcid(record):
```

Comment:
> As an example ...

IMHO we don't want to show Zenodo or INSPIRE data model here. MARC21 is neutral in digital library world.